docs/completeExamples/completeExamples.md

```diff
@@ -0,0 +1,44 @@
+# Complete Examples
+A number of examples of complete events are provided below, in order to illustrate use of the schema.
+
+N.B. All entities including organisations, persons, devices and applications are imaginary.
+All details (e.g. MAC Address, Name, IP Address) are similarly imaginary and any relationship to any 
+real entities is entirely coincidental!
+
+In some places ```<UserDetails>``` have been abbreviated for clarity using an elipsis (...).
+
+In a real-world situation, a design decision would be needed - when to leave the event more concise using
+only an ID and when to join with additional information.  Data such as user details
```

Comment:
Also when to do that enrichment e.g. could be done by the source system or at a later stage